common/components/componentsBySection/MyProjects/MyProjectCard.jsx

```diff
@@ -61,18 +61,40 @@ class MyProjectCard extends React.PureComponent<Props, State> {
   }
 
   _renderProjectStatus(): React$Node {
-    const header: string = this.state.isOwner
-      ? "Project Status"
-      : "Volunteer Status";
-    const status: string = this.state.isOwner
-      ? this.props.project.isApproved
-        ? "Published"
-        : "Unpublished"
-      : !this.props.project.isApproved
-      ? "Awaiting Approval"
-      : this.props.project.isUpForRenewal
-      ? "Expires on " + moment(this.props.project.projectedEndDate).format("l")
-      : "Active";
+
+    let header = ""
+
+    if (this.state.isOwner) {
+      header = "Project Status";
+    }
+
+    else {
+      header = "Volunteer Status";
+    }
+
+    let status = "";
+
+    if (this.state.isOwner) {
+      if (this.props.project.isApproved) {
+        status = "Published";
+      }
+      else {
+        status = "Unpublished";
+      }
+    }
+
+    else if (this.props.project.isUpForRenewal) {
+      status = "Expires on " + moment(this.props.project.projectedEndDate).format("l");
+    }
+
+    else if (!this.state.isOwner && !this.props.project.isApproved) {
+      status = "Pending";
+    }
+
+    else {
+      status = "Active";
+    }
```

Comment:
```suggestion
    if (this.state.isOwner) {
      if (this.props.project.isApproved) {
        status = "Published";
      } else {
        status = "Unpublished";
      }
    } else if (this.props.project.isUpForRenewal) {
      status = "Expires on " + moment(this.props.project.projectedEndDate).format("l");
    } else if (!this.state.isOwner && !this.props.project.isApproved) {
      status = "Pending";
    } else {
      status = "Active";
    }
```
It's easier to see that these code blocks are part of a single conditional check if they're attached to each other rather than on separate lines.